app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java

```diff
@@ -206,7 +206,9 @@ private void renderSessions(List<SessionViewModel> adjustedSessionViewModels) {
 
         adapter.reset(adjustedSessionViewModels);
 
-        binding.txtDate.setText(adjustedSessionViewModels.get(0).getFormattedDate());
+        if (TextUtils.isEmpty(binding.txtDate.getText())) {
```

Comment:
By the way, it's better to set text by DataBinding.
I'll try to improve on another PR!